ament_tools/commands/ament.py

```diff
@@ -30,6 +31,8 @@ def main(sysargs=None):
     # Assign sysargs if not set
     sysargs = sys.argv[1:] if sysargs is None else sysargs
 
+    os.environ['AMENT_TOOLS'] = 'true'
```

Comment:
Maybe set it to `1` to not give the wrong expectation that setting it to `false` be the opposite?